client/src/components/ProductDetailPage.jsx

```diff
@@ -28,7 +28,7 @@ class ProductDetailPage extends React.Component {
       products: {
         // productId: { ... },
       },
-      yourOutfit: [],
+      yourOutfit: JSON.parse(localStorage.getItem('atelier-your-outfit')) || [],
```

Comment:
Is there a compelling reason to store the outfit in localStorage vs. in memory? Is this a substitute for sessioncookie/user state approach?

Reply:
This just allows it to persist across sessions. Nothing we need to do, but it's neat to have it